templates/pages/about.html

```diff
@@ -8,27 +8,59 @@
 
 
 	<div class="jumbotron">
-		<p>We, CodeCatz, are learning to code, beause we can and because we want to! </p>
-		<p>CodeCatz is a group of programming kittens and several slightly more experienced cats. Kittens want to learn coding, cats are here to help them. We started with Python and web applications, we'll continue with whatnots.</p>
-		<p> Who knows where we will end! </p>
+		<h2 class="about">About CodeCatz</h2>
+		<p>
```

Comment:
no, that's not right... just remove them for now... this can (and should) be fixed in styling, not in the markup